Cargo.toml

```diff
@@ -26,6 +26,7 @@ valgrind = []
 [dependencies]
 libc = "0.2.11"
 librocksdb_sys = { path = "librocksdb_sys" }
+librocksdb_cloud_sys = { path = "librockdb_sys" }
```

Comment:
typo: librocksdb_sys

Reply:
Aah, yeah sorry about that, but I think you mean `librocksdb_cloud_sys`